lib/faraday/request/multipart.rb

```diff
@@ -61,6 +61,14 @@ def create_multipart(env, params)
         body
       end
 
+      def part(boundary, key, value)
+        if value.is_a?(Faraday::Request::PartParams)
```

Comment:
What do you think about changing this to check if `value` responds to `to_part`? It'll make this more generic, giving users the ability to add their own "Part" implementation.

Reply:
great idea, I'm buying

Reply:
✅